test/e2e/etcd_failure.go

```diff
@@ -121,7 +108,7 @@ func checkExistingRCRecovers(f Framework) {
 	rcSelector := labels.Set{"name": "baz"}.AsSelector()
 
 	By("deleting pods from existing replication controller")
-	expectNoError(wait.Poll(time.Millisecond*500, time.Second*30, func() (bool, error) {
+	expectNoError(wait.Poll(time.Second*5, time.Second*60, func() (bool, error) {
```

Comment:
You could probably leave the retry period at 500ms.  I don't feel terribly strongly about that though.